src/Forms/Shared/Samples/Symbology/CustomDictionaryStyle/CustomDictionaryStyle.xaml.cs

```diff
@@ -30,45 +32,83 @@ public partial class CustomDictionaryStyle : ContentPage
+                // Create a new map with a topographic basemap.
+                Map map = new Map(BasemapStyle.ArcGISTopographic);
 
                 // Create the restaurants layer and add it to the map.
-                FeatureLayer restaurantLayer = new FeatureLayer(_restaurantUri);
-                map.OperationalLayers.Add(restaurantLayer);
+                _restaurantLayer = new FeatureLayer(_restaurantUri);
+                map.OperationalLayers.Add(_restaurantLayer);
 
                 // Load the feature table for the restaurants layer.
-                await restaurantLayer.FeatureTable.LoadAsync();
+                await _restaurantLayer.FeatureTable.LoadAsync();
 
-                // Open the custom style file.
-                DictionarySymbolStyle restaurantStyle = await DictionarySymbolStyle.CreateFromFileAsync(_stylxPath);
+                // Open the local custom style file.
+                _localRestaurantStyle = await DictionarySymbolStyle.CreateFromFileAsync(_stylxPath);
 
-                // Create the dictionary renderer with the style file and the field overrides.
-                DictionaryRenderer dictRenderer = new DictionaryRenderer(restaurantStyle);
+                // Open a portal item containing a dictionary symbol style.
+                ArcGISPortal portal = await ArcGISPortal.CreateAsync();
+                PortalItem portalItem = await PortalItem.CreateAsync(portal, "adee951477014ec68d7cf0ea0579c800");
 
-                // Set the restaurant layer renderer to the dictionary renderer.
-                restaurantLayer.Renderer = dictRenderer;
+                // Open the portal custom style file.
+                _restaurantStyleFromPortal = await DictionarySymbolStyle.OpenAsync(portalItem);
 
                 // Set the map's initial extent to that of the restaurants.
-                map.InitialViewpoint = new Viewpoint(restaurantLayer.FullExtent);
+                map.InitialViewpoint = new Viewpoint(_restaurantLayer.FullExtent);
 
                 // Set the map to the map view.
                 MyMapView.Map = map;
+
+                // Create the dictionary renderer with the style file and the field overrides.
+                DictionaryRenderer dictionaryRenderer = new DictionaryRenderer(_localRestaurantStyle);
+
+                // Set the restaurant layer renderer to the dictionary renderer.
+                _restaurantLayer.Renderer = dictionaryRenderer;
             }
             catch (Exception ex)
             {
-                Console.WriteLine(ex.Message);
+                await Application.Current.MainPage.DisplayAlert("Error", ex.Message, "OK");
             }
         }
+
+        private void LocalStyleButtonClicked(object sender, EventArgs e)
+        {
+            // Create the dictionary renderer with the style file and the field overrides.
+            DictionaryRenderer dictionaryRenderer = new DictionaryRenderer(_localRestaurantStyle);
```

Comment:
To make the sample code easier to comprehend, I think it would be better to have separate `LoadLocalStyle` and `LoadWebStyle` methods. Those two steps are be the core of the sample that users are interested in, so it will be good to isolate that from the button and map setup overhead.